app/src/main/java/org/sagebionetworks/research/mpower/tracking/view_model/logs/NoteLog.java

```diff
@@ -0,0 +1,25 @@
+package org.sagebionetworks.research.mpower.tracking.view_model.logs;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+/**
+ * Represents a log which stores a note. A Note is any String that the user cna input to describe the log in case they
+ * want to enter information other than what the UX allows.
+ */
+public interface NoteLog extends TrackingItemLog {
```

Comment:
In Kotlin, you can define multiple interfaces in a single file.  Seems like that would really help readability for architecture like this where you have many light-weight interfaces.